docs/static/core-plugins/codecs/java_dots.asciidoc

```diff
@@ -1,22 +1,19 @@
-:plugin: jdots
+:plugin: java_dots
```

Comment:
The tag for this codec is actually `jdots`. That may be a bad choice on my part and perhaps it should be changed, but if it's listed as `Java_dots` in the docs, I don't think there's any other indication that it must be referred to as `jdots` in the Logstash pipeline definition.